cluster/gce/gci/README.md

```diff
@@ -21,12 +21,36 @@ Policy](https://cloud.google.com/container-optimized-os/docs/resources/support-p
 Container-Optimized OS images are used by kubernetes [End-to-End tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md) and
 [Node End-to-End tests](https://github.com/kubernetes/community/tree/master/contributors/devel/sig-node). To see current
 active releases, please refer to COS's [Release
-Notes](https://cloud.google.com/container-optimized-os/docs/release-notes). When
-choose images for testing, the latest LTS images is suggested for running
-presubmit, postsubmit or periodics tests since the image is stable and has
-latest bug and security fixes. For testing requiring new features or continuous integration,
-the latest dev or beta or stable channel can be used.
-are needed, the latest LTS images or stables are preferred.
+Notes](https://cloud.google.com/container-optimized-os/docs/release-notes).
+
+### How to choose image for your test
+
+  * To run presubmit, postsubmit or periodic tests, the latest LTS images are
+    preferred. If tests need two images, you can use the latest two LTS images.
+    For example, 81 and 77. LTS images are stable and usually include latest bug and security fix.
```

Comment:
The for example, doesn't really help. It's not going to be 81 and 77 in the future. I don't thin we need an example at all.